src/index.ts

```diff
@@ -102,7 +102,7 @@ export { extractFiles } from "./exchange-connector/exchangeDirectoryParser";
 export { RestApi } from "./exchange-connector/exchangeTypes";
 export { ramlToolLogger } from "./logger";
 
-export { model, AMF, plugins } from "amf-client-js";
+export { default as amf } from "amf-client-js";
```

Comment:
This breaks the SDK. I do agree overall with Will, but I'd like
```suggestion
export amf, { model } from "amf-client-js";
```
It makes sense to me export model at the top level since it is also the model we're using.

Reply:
I updated all the references in the SDK so it shouldn't break.

Reply:
I'll push the changes in commerce-sdk once these are approved.

Reply:
I couldn't find a way to do that successfully in Typescript. @bredmond-sf @wjhsf I think we will have to live with this.

Reply:
```suggestion
export * as amf from "amf-client-js";
export { model } from "amf-client-js";
```
I think this should do it. I don't think there's a way to consolidate that into one statement.

Reply:
Thanks Will for the suggestion.

Reply:
Please take a look and hit approve if everything looks alright.